### PR TITLE
DAT-18309 DevOps :: Extensions are creating 2 tags

### PR DESCRIPTION
**Pull request by jandroav**
https://datical.atlassian.net/browse/DAT-18309

### CODE REVIEW

File: .github/workflows/extension-release-published.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ on:
       deployToMavenCentral:
         description: 'Specify it if you want to deploy to maven'
         required: false
-        default: 'true'
+        default: true
```
Comment (jandroav) [inline .github/workflows/extension-release-published.yml:19]: this was failing @sayaliM0412  
image

File: .github/release-drafter.yml [hunk 2]
```diff
@@ -51,5 +51,3 @@ template: |
   ## Changes
 
   $CHANGES
-
-  **Full Changelog**: https://github.com/$OWNER/$REPOSITORY/compare/$PREVIOUS_TAG...$REPOSITORY-v$RESOLVED_VERSION
```
Comment (jandroav) [inline .github/release-drafter.yml:55]: Same as liquibase/liquibase so it ca be auto generated

**Comment by jandroav** [comment]
Do we want to change all `@vX.X.X` workflow references to `@main` and maintain `v0.7.8` synched with `main` for the existing extensions extensions transition to `@main`? @jnewton03 @filipelautert @sayaliM0412

**Comment by sayaliM0412** [comment]
> Do we want to change all `@vX.X.X` workflow references to `@main` and maintain `v0.7.8` synched with `main` for the existing extensions extensions transition to `@main`? @jnewton03 @filipelautert @sayaliM0412

Sounds good for now. I ended up updating the `v.0.7.8` for previous changes as well.

**Review by jnewton03 — APPROVED** [review]

**Pull request merged**